### ISSUE TITLE
Cannot start lem again after exit

### ISSUE THREAD

**Issue by rpx99** [lem-project/lem#1097]
Once I exit Lem and fall back on the sbcl repl I cannot start lem again (sdl2).
```
(:LEM-SDL2)
* (lem:lem)
NIL
* (lem:lem)
NIL
```

I need to quit sbcl and quickload again to be able to start lem again.

**Comment by Sasanidas** [comment]
Indeed, that maybe a problem, Lem is not intended to be launch using the REPL, you can compile the lem to an image using the make command:
`make build-sdl2`

https://lem-project.github.io/lem-page/installation/getting-started/#with-make

**Comment by rpx99** [comment]
Hm, but why ? :-)

**Comment by Sasanidas** [comment]
I'm not sure, but it's probably to some initialization configuration :thinking:

**Comment by cxxxr** [comment]
Thank you for the report.
This problem has been neglected, but I want to fix.

**Comment by cxxxr** [comment]
maybe, It have fixed.
